pkgs/applications/audio/aaxtomp3/default.nix

```diff
@@ -36,17 +27,51 @@ stdenv.mkDerivation rec {
     hash = "sha256-7a9ZVvobWH/gPxa3cFiPL+vlu8h1Dxtcq0trm3HzlQg=";
   };
 
-  dontBuild = false;
+  # use whitespace to show osh arithmetic is not file redirection
```

Comment:
Do we really need a patch for resholve to work? Sounds to me like bug we should report.

Reply:
yeah, it is 100% an upstream bug; see the commit message, when I figure out how to reproduce the issue with osh, I will report it upstream

Reply:
Might be able to do something like `nix-shell -p oil --command "osh -n AAXtoMP3"`?

Reply:
[not quite](https://github.com/NixOS/nixpkgs/files/10369836/77l4hw337g8c7zfczzs0dsww8sj96hi9-aaxtomp3-1.3.log), let me test it out with some different versions

Reply:
I'm invoking osh from oil 0.12.0 (the version in nixpkgs master is 0.12.9).

The error message I saw was committed more-recently than the oil resholve is currently using, so I guess the message has changed but it would still be an error.

Reply:
yep, using 0.8.12 gives the error we want; I will report this upstream now:
```console
[user@localhost:~/src/aaxtomp3]$ osh -n AAXtoMP3
    for (( n=0; n<(20-(percentage/5)); n++ )) ; do progressbar="$progressbar "; done
                       ^~~~~~~~~~~~
AAXtoMP3:204: Expected ) in function definition
```